lib/utils/validator-extras.js

```diff
@@ -77,9 +77,8 @@ function extendModelValidations(modelInstance) {
 }
 exports.extendModelValidations = extendModelValidations;
 
-// Deprecate this.
-validator.notNull = function() {
-  throw new Error('Warning "notNull" validation has been deprecated in favor of Schema based "allowNull"');
+validator.notNull = function(val) {
+  return [null, undefined].indexOf(val) === -1;
```

Comment:
Since v5 targets NodeJS 6+ it's now possible to `return [null, undefined].includes(val)`

Reply:
Thanks, I just copied this code from old PR, will change